src/components/Modal/AddModal.tsx

```diff
@@ -0,0 +1,62 @@
+import styles from "./modal.module.css";
+import ModalCloseButton from "../common/Modal/ModalButtons/ModalCloseButton";
+import ModalResetButton from "../common/Modal/ModalButtons/ModalResetButton";
+import ModalSubmitButton from "../common/Modal/ModalButtons/ModalSubmitButton";
+import ModalForm from "../common/Modal/ModalForm/ModalForm";
+import { AddModalType } from "../../types/modal/addModal.type";
+import { useState } from "react";
+
+function AddModal({ toggle, handleCloseButton }: AddModalType) {
+  const [title, setTitle] = useState("");
+  const [release, setRelease] = useState("");
+  const [url, setUrl] = useState("");
+  const [rating, setRating] = useState("");
+  const [runtime, setRuntime] = useState("");
+  const [overview, setOverview] = useState("");
+  const [genres, setGenres] = useState<string[]>([]);
+
+  const handleReset = () => {
+    setTitle("");
+    setRelease("");
+    setUrl("");
+    setRating("");
+    setRuntime("");
+    setOverview("");
+    setGenres([]);
+  };
+
+  const handleSubmit = () => {
+    //  TODO: Implement submit func
+  };
+
+  return (
+    <>
+      {toggle && (
```

Comment:
It would be better to use [portals](https://reactjs.org/docs/portals.html) on functional components